packages/core/test-helpers/core-test-helpers-kbn-server/src/create_root.ts

```diff
@@ -284,8 +284,10 @@ export function createTestServers({
         password: kibanaServerTestUser.password,
       };
     },
-    startKibana: async () => {
-      const root = createRootWithCorePlugins(kbnSettings);
+    startKibana: async (abortSignal?: AbortSignal) => {
+      const root = createRootWithCorePlugins(kbnSettings, cliArgs);
+
+      abortSignal?.addEventListener('abort', async () => await root.shutdown());
```

Comment:
I needed this so I could stop Kibana when left hanging waiting for a valid ES connection